tsconfig.json

```diff
@@ -7,13 +8,15 @@
         "sourceMap": true,
         "target": "ES6",
         "declaration": true,
-        "moduleResolution": "node"
+        "moduleResolution": "node",
+        "outDir": "lib"
```

Comment:
I usually prefer `dist/` since this is not really a library.

Reply:
Updated to dist